### ISSUE TITLE
Fails to download game due to memory cache being too small

### ISSUE THREAD

**Issue by mks-h** [Heroic-Games-Launcher/HeroicGamesLauncher#2732]
### Describe the bug

When trying to download Fallout New Vegas, I get the following error after downloading 11% of it, and then it stops. Subsequent attempts to download it, obviously, fail in the same way.
```
Legendary (Epic Games)
Error:
An error has occurred! Try to Logout and Login on your Epic account.
[Core] INFO: Trying to re-use existing login session...[cli] INFO: Preparing download for "Fallout: New Vegas" (5daeb974a22a435988892319b3a4f476)...[Core] WARNING: Unable to download manifest from "download.epicgames.com" (status: 403), trying next URL...[Core] WARNING: Unable to download manifest from "download2.epicgames.com" (status: 403), trying next URL...[Core] WARNING: Unable to download manifest from "download3.epicgames.com" (status: 403), trying next URL...[Core] WARNING: Unable to download manifest from "download4.epicgames.com" (status: 403), trying next URL...[Core] INFO: Parsing game manifest...[Core] INFO: Install path: /home/mks/Games/Heroic/FalloutNewVegas
[Core] INFO: Selected CDN: download.epicgames.com (https)[DLM] INFO: Found previously interrupted download. Download will be resumed if possible.[DLM] INFO: Skipping 6 files based on resume data.Traceback (most recent call last):
File "legendary/cli.py", line 3061, in <module>File "legendary/cli.py", line 2978, in main
File "legendary/cli.py", line 936, in install_game
File "legendary/core.py", line 1453, in prepare_download
File "legendary/downloader/mp/manager.py", line 395, in run_analysis
MemoryError: Current shared memory cache is smaller than required: 2048.0 MiB < 4902.0 MiB. Try running legendary with "--enable-reordering" to reduce memory usage, or use "--max-shared-memory 4934" to increase the limit.
[653] Failed to execute script 'cli' due to unhandled exception!
```

### Add logs

```text
(22:05:39) INFO:    [Legendary]:        Refreshing library...
(22:05:39) INFO:    [Connection]:       Connectivity: check-online
(22:05:39) INFO:    [Connection]:       Pinging external endpoints
(22:05:39) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(22:05:39) INFO:    [Legendary]:        Legendary location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(22:05:39) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(22:05:39) INFO:    [Gog]:              GOGDL location: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(22:05:39) WARNING: [Backend]:          Failed to register protocol with OS.
(22:05:39) INFO:    [Connection]:       Connectivity: online
(22:05:39) INFO:    [Backend]:          User Not Found, removing it from Store
(22:05:39) INFO:    [Gog]:              Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/gogdl --auth-config-path /home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/gog_store/auth.json --version
(22:05:39) INFO:    [Backend]:          AreWeAntiCheatYet data downloaded
(22:05:40) INFO:    [Backend]:          Loading Screen Ready
(22:05:40) INFO:    [Backend]:          Frontend Ready
(22:05:40) INFO:    [Backend]:          

Heroic Version: 2.7.1 Eustass Kid
Legendary Version:  0.20.32 Dark Energy (hotfix #6)
GOGdl Version: 0.7.1

Electron Version: 23.1.4
Chrome Version: 110.0.5481.192
NodeJS Version: 18.12.1

OS: Freedesktop SDK KERNEL: 6.2.15-300.fc38.x86_64 ARCH: x64
CPU: Intel Core™ i5-6200U @2.3 GOVERNOR: powersave
RAM: Total: 7.63 GiB Available: 3.53 GiB
GRAPHICS: GPU0: Skylake GT2 [HD Graphics 520]  VRAM: 256MB GPU1: GM108M [GeForce 940MX]  VRAM: 32MB 
PROTOCOL: wayland

(22:05:40) INFO:    [Backend]:          Checking for current version changelog
(22:05:41) INFO:    [Winetricks]:       Downloading Winetricks
(22:05:45) INFO:    [Backend]:          Starting the Download Queue
(22:06:19) INFO:    [Legendary]:        Logging in: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary auth --code <redacted>
(22:06:20) INFO:    [Frontend]:         Refreshing Library
(22:06:20) INFO:    [Legendary]:        Refreshing library...
(22:06:20) INFO:    [Legendary]:        Refreshing Epic Games...
(22:06:20) INFO:    [Legendary]:        Game list updated, got 0 games & DLCs
(22:06:20) INFO:    [Frontend]:         Called Login
(22:06:20) INFO:    [Frontend]:         No cache found, getting data from legendary...
(22:06:20) INFO:    [Legendary]:        Refreshing library...
(22:06:20) INFO:    [Legendary]:        Refreshing Epic Games...
(22:06:21) INFO:    [Legendary]:        Game list updated, got 0 games & DLCs
(22:06:21) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:06:21) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:06:23) INFO:    [Frontend]:         Refreshing Library
(22:06:23) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(22:06:23) INFO:    [Legendary]:        Refreshing library...
(22:06:23) INFO:    [Legendary]:        Refreshing Epic Games...
(22:06:23) INFO:    [Legendary]:        Game list updated, got 8 games & DLCs
(22:06:23) INFO:    [Frontend]:         No cache found, getting data from legendary...
(22:06:23) INFO:    [Legendary]:        Refreshing library...
(22:06:23) INFO:    [Legendary]:        Refreshing Epic Games...
(22:06:23) INFO:    [Legendary]:        Game list updated, got 8 games & DLCs
(22:06:23) INFO:    [Frontend]:         Force Update
(22:06:23) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:06:23) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:08:58) INFO:    [Legendary]:        Getting more details with 'legendary info'
(22:08:58) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(22:08:58) DEBUG:   [Backend]:          {"free":344503300096,"diskSize":498387124224,"message":"320.84 GiB / 464.16 GiB","validPath":true}
(22:08:58) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary info 5daeb974a22a435988892319b3a4f476 --platform Windows --json
(22:09:03) DEBUG:   [Backend]:          {"free":344498958336,"diskSize":498387124224,"message":"320.84 GiB / 464.16 GiB","validPath":true}
(22:09:16) INFO:    [Backend]:          Writing config for 5daeb974a22a435988892319b3a4f476
(22:09:16) INFO:    [Backend]:          Changed config: winePrefix from /home/mks/Games/Heroic/Prefixes/default to /home/mks/Games/Heroic/Prefixes/default/Fallout New Vegas
(22:09:16) INFO:    [Backend]:          Changed config: wineCrossoverBottle from undefined to 
(22:09:16) DEBUG:   [Legendary]:        Using cached install info
(22:09:16) INFO:    [DownloadManager]:  Fallout: New Vegas  was added to the download queue.
(22:09:16) INFO:    [Backend]:          Preventing machine to sleep
(22:09:16) DEBUG:   [Legendary]:        Using cached install info
(22:09:16) INFO:    [Legendary]:        Installing 5daeb974a22a435988892319b3a4f476: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y
(22:09:16) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:09:20) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y --max-shared-memory 5000
(22:09:20) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:09:24) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0%/0.00MB/00:00:00 Down: 0MB/s / Disk: 0MB/s
(22:09:30) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.01%/0.85MB/10:15:05 Down: 0MB/s / Disk: 0MB/s
(22:09:31) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.01%/0.85MB/11:57:17 Down: 0MB/s / Disk: 0MB/s
(22:09:55) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.21%/29.71MB/09:36:03 Down: 0MB/s / Disk: 0MB/s
(22:10:28) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.48%/68.41MB/09:12:02 Down: 2.97MB/s / Disk: 1MB/s
(22:10:45) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.61%/86.28MB/08:58:51 Down: 0MB/s / Disk: 0MB/s
(22:11:00) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.72%/102.10MB/09:03:46 Down: 0MB/s / Disk: 0MB/s
(22:11:21) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.89%/126.80MB/09:18:49 Down: 1.97MB/s / Disk: 0MB/s
(22:11:39) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 1.04%/147.48MB/09:02:37 Down: 1.98MB/s / Disk: 1MB/s
(22:13:39) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 1.99%/281.48MB/09:02:42 Down: 1.99MB/s / Disk: 0MB/s
(22:13:46) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 2.04%/289.41MB/09:09:50 Down: 1MB/s / Disk: 0MB/s
(22:13:49) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 2.05%/291.39MB/08:59:16 Down: 0MB/s / Disk: 0MB/s
(22:13:54) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 2.1%/297.36MB/09:03:56 Down: 1.99MB/s / Disk: 0MB/s
(22:15:38) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 2.91%/413.17MB/08:57:24 Down: 1MB/s / Disk: 1MB/s
(22:15:54) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 3.04%/431.09MB/09:00:56 Down: 2.98MB/s / Disk: 0MB/s
(22:18:49) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 4.41%/625.60MB/08:58:57 Down: 0.99MB/s / Disk: 0MB/s
(22:19:19) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 4.66%/660.27MB/08:54:49 Down: 0.97MB/s / Disk: 3MB/s
(22:20:31) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 5.22%/739.82MB/08:55:51 Down: 0.99MB/s / Disk: 0MB/s
(22:21:10) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 5.53%/783.57MB/08:52:57 Down: 2MB/s / Disk: 4MB/s
(22:21:48) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 5.83%/827.25MB/08:52:49 Down: 1.98MB/s / Disk: 3MB/s
(22:27:42) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 8.6%/1219.95MB/08:19:42 Down: 0MB/s / Disk: 0MB/s
(22:28:00) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 8.75%/1240.14MB/08:17:57 Down: 1.61MB/s / Disk: 1MB/s
(22:28:30) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 8.98%/1273.38MB/08:15:48 Down: 1.62MB/s / Disk: 0MB/s
(22:28:47) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.12%/1292.74MB/08:13:01 Down: 0.67MB/s / Disk: 0MB/s
(22:29:12) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.31%/1320.31MB/08:10:04 Down: 0MB/s / Disk: 0MB/s
(22:29:13) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.33%/1322.46MB/08:10:29 Down: 2.15MB/s / Disk: 0MB/s
(22:29:21) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.38%/1330.08MB/08:08:26 Down: 0.59MB/s / Disk: 0MB/s
(22:29:34) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.47%/1342.37MB/08:07:46 Down: 0MB/s / Disk: 0MB/s
(22:29:35) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.49%/1345.12MB/08:07:28 Down: 2.74MB/s / Disk: 2MB/s
(22:29:58) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.67%/1371.70MB/08:05:19 Down: 1.47MB/s / Disk: 1MB/s
(22:30:14) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.8%/1389.01MB/08:04:51 Down: 1MB/s / Disk: 1MB/s
(22:30:28) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.9%/1403.95MB/08:05:16 Down: 0MB/s / Disk: 0MB/s
(22:30:35) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.96%/1411.83MB/08:06:18 Down: 0.92MB/s / Disk: 0MB/s
(22:30:36) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 9.96%/1412.83MB/08:05:42 Down: 0.99MB/s / Disk: 3MB/s
(22:31:00) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 10.16%/1440.33MB/08:04:59 Down: 1.52MB/s / Disk: 0MB/s
(22:31:01) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 10.17%/1441.65MB/08:04:04 Down: 1.32MB/s / Disk: 4MB/s
(22:31:28) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 10.37%/1469.77MB/08:01:59 Down: 0.92MB/s / Disk: 0MB/s
(22:31:38) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 10.44%/1480.80MB/08:00:37 Down: 0.74MB/s / Disk: 1MB/s
(22:31:53) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 10.56%/1496.87MB/08:00:28 Down: 1.5MB/s / Disk: 0MB/s
(22:32:16) INFO:    [Backend]:          Trying to kill 5daeb974a22a435988892319b3a4f476
(22:32:16) INFO:    [Backend]:          Killed 5daeb974a22a435988892319b3a4f476
(22:32:16) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:32:16) INFO:    [Legendary]:        Abort command "/app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y --max-shared-memory 5000"
(22:32:16) WARNING: [DownloadManager]:  Installation of 5daeb974a22a435988892319b3a4f476 aborted!
(22:32:16) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 added to download manager finished.
(22:32:16) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:32:16) INFO:    [Frontend]:         Refreshing Library
(22:32:16) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(22:32:16) INFO:    [Legendary]:        Refreshing library...
(22:32:16) INFO:    [Legendary]:        Refreshing Epic Games...
(22:32:16) INFO:    [Legendary]:        Game list updated, got 9 games & DLCs
(22:32:16) INFO:    [Frontend]:         Force Update
(22:32:17) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:32:17) DEBUG:   [Legendary]:        Using cached install info
(22:32:17) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(22:32:17) DEBUG:   [Backend]:          {"free":342810599424,"diskSize":498387124224,"message":"319.27 GiB / 464.16 GiB","validPath":true}
(22:32:17) DEBUG:   [Backend]:          {"free":342810599424,"diskSize":498387124224,"message":"319.27 GiB / 464.16 GiB","validPath":true}
(22:32:19) INFO:    [Backend]:          Writing config for 5daeb974a22a435988892319b3a4f476
(22:32:19) DEBUG:   [Legendary]:        Using cached install info
(22:32:19) INFO:    [DownloadManager]:  Fallout: New Vegas  was added to the download queue.
(22:32:19) INFO:    [Backend]:          Preventing machine to sleep
(22:32:20) DEBUG:   [Legendary]:        Using cached install info
(22:32:20) INFO:    [Legendary]:        Installing 5daeb974a22a435988892319b3a4f476: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --skip-dlcs --skip-sdl -y
(22:32:20) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:32:23) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --skip-dlcs --skip-sdl -y --max-shared-memory 5000
(22:32:23) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:32:27) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0%/0.00MB/00:00:00 Down: 0MB/s / Disk: 0MB/s
(22:32:33) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.03%/3.74MB/59:45:23 Down: 3.73MB/s / Disk: 0.57MB/s
(22:32:47) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.14%/18.71MB/08:16:55 Down: 1.87MB/s / Disk: 2MB/s
(22:32:49) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.15%/21.00MB/08:24:30 Down: 0MB/s / Disk: 0MB/s
(22:33:16) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.36%/51.12MB/06:56:44 Down: 1.55MB/s / Disk: 2MB/s
(22:34:49) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 1.09%/154.49MB/06:52:46 Down: 0MB/s / Disk: 0MB/s
(22:34:50) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 1.1%/155.82MB/06:55:41 Down: 1.33MB/s / Disk: 0MB/s
(22:34:51) INFO:    [ExtraGameInfo]:    Getting ExtraGameInfo data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting PCGamingWiki data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting HowLongToBeat data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting GamesDB data for Fallout New Vegas
(22:34:51) DEBUG:   [Legendary]:        Using cached install info
(22:34:51) INFO:    [ExtraGameInfo]:    Getting ExtraGameInfo data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting PCGamingWiki data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting HowLongToBeat data for Fallout New Vegas
(22:34:51) INFO:    [ExtraGameInfo]:    Getting GamesDB data for Fallout New Vegas
(22:34:51) DEBUG:   [Legendary]:        Using cached install info
(22:34:51) DEBUG:   [Legendary]:        Using cached install info
(22:34:51) DEBUG:   [Legendary]:        Using cached install info
(22:34:53) INFO:    [Legendary]:        Getting Info from Epic API
(22:34:54) INFO:    [Legendary]:        Getting Info from Epic API
(22:35:14) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Fallout New Vegas
(22:35:14) DEBUG:   [Legendary]:        Using cached install info
(22:35:14) INFO:    [ExtraGameInfo]:    Using cached ExtraGameInfo data for Fallout New Vegas
(22:35:14) DEBUG:   [Legendary]:        Using cached install info
(22:35:14) DEBUG:   [Legendary]:        Using cached install info
(22:35:14) DEBUG:   [Legendary]:        Using cached install info
(22:35:18) INFO:    [Backend]:          Trying to kill 5daeb974a22a435988892319b3a4f476
(22:35:18) INFO:    [Backend]:          Killed 5daeb974a22a435988892319b3a4f476
(22:35:18) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:35:18) INFO:    [Legendary]:        Abort command "/app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --skip-dlcs --skip-sdl -y --max-shared-memory 5000"
(22:35:18) WARNING: [DownloadManager]:  Installation of 5daeb974a22a435988892319b3a4f476 aborted!
(22:35:18) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 added to download manager finished.
(22:35:18) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:35:18) INFO:    [Frontend]:         Refreshing Library
(22:35:18) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(22:35:18) INFO:    [Legendary]:        Refreshing library...
(22:35:18) INFO:    [Legendary]:        Refreshing Epic Games...
(22:35:18) INFO:    [Legendary]:        Game list updated, got 9 games & DLCs
(22:35:18) INFO:    [Backend]:          Stopping Power Saver Blocker
(22:35:18) DEBUG:   [Legendary]:        Using cached install info
(22:35:18) DEBUG:   [Legendary]:        Using cached install info
(22:35:18) DEBUG:   [Legendary]:        Using cached install info
(22:35:18) DEBUG:   [Legendary]:        Using cached install info
(22:35:18) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:35:20) DEBUG:   [Legendary]:        Using cached install info
(22:35:20) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(22:35:20) DEBUG:   [Backend]:          {"free":343246729216,"diskSize":498387124224,"message":"319.67 GiB / 464.16 GiB","validPath":true}
(22:35:20) DEBUG:   [Backend]:          {"free":343246729216,"diskSize":498387124224,"message":"319.67 GiB / 464.16 GiB","validPath":true}
(22:35:21) INFO:    [Backend]:          Writing config for 5daeb974a22a435988892319b3a4f476
(22:35:21) DEBUG:   [Legendary]:        Using cached install info
(22:35:21) INFO:    [DownloadManager]:  Fallout: New Vegas  was added to the download queue.
(22:35:21) DEBUG:   [Legendary]:        Using cached install info
(22:35:21) DEBUG:   [Legendary]:        Using cached install info
(22:35:22) DEBUG:   [Legendary]:        Using cached install info
(22:35:22) INFO:    [Legendary]:        Installing 5daeb974a22a435988892319b3a4f476: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --skip-dlcs --skip-sdl -y
(22:35:22) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:35:22) DEBUG:   [Legendary]:        Using cached install info
(22:35:22) DEBUG:   [Legendary]:        Using cached install info
(22:35:25) INFO:    [Backend]:          Trying to kill 5daeb974a22a435988892319b3a4f476
(22:35:25) INFO:    [Backend]:          Killed 5daeb974a22a435988892319b3a4f476
(22:35:25) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:35:25) INFO:    [Legendary]:        Abort command "/app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --skip-dlcs --skip-sdl -y"
(22:35:25) WARNING: [DownloadManager]:  Installation of 5daeb974a22a435988892319b3a4f476 aborted!
(22:35:26) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 added to download manager finished.
(22:35:26) INFO:    [DownloadManager]:  5daeb974a22a435988892319b3a4f476 removed from download manager.
(22:35:26) INFO:    [Frontend]:         Refreshing Library
(22:35:26) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(22:35:26) INFO:    [Legendary]:        Refreshing library...
(22:35:26) INFO:    [Legendary]:        Refreshing Epic Games...
(22:35:26) INFO:    [Legendary]:        Game list updated, got 9 games & DLCs
(22:35:26) INFO:    [Backend]:          Stopping Power Saver Blocker
(22:35:26) DEBUG:   [Legendary]:        Using cached install info
(22:35:26) DEBUG:   [Legendary]:        Using cached install info
(22:35:26) DEBUG:   [Legendary]:        Using cached install info
(22:35:26) DEBUG:   [Legendary]:        Using cached install info
(22:35:26) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(22:35:26) DEBUG:   [Backend]:          {"free":343180169216,"diskSize":498387124224,"message":"319.61 GiB / 464.16 GiB","validPath":true}
(22:35:26) DEBUG:   [Backend]:          {"free":343180169216,"diskSize":498387124224,"message":"319.61 GiB / 464.16 GiB","validPath":true}
(22:35:26) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
(22:35:28) INFO:    [Backend]:          Writing config for 5daeb974a22a435988892319b3a4f476
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:28) INFO:    [DownloadManager]:  Fallout: New Vegas  was added to the download queue.
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:28) INFO:    [Legendary]:        Installing 5daeb974a22a435988892319b3a4f476: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y
(22:35:28) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:28) DEBUG:   [Legendary]:        Using cached install info
(22:35:32) INFO:    [Legendary]:        Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y --max-shared-memory 5000
(22:35:32) DEBUG:   [Legendary]:        Logging to file "/home/mks/.var/app/com.heroicgameslauncher.hgl/config/heroic/GamesConfig/5daeb974a22a435988892319b3a4f476.log"
(22:35:40) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.01%/1.43MB/49:49:31 Down: 0.67MB/s / Disk: 0MB/s
(22:35:51) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.12%/15.95MB/07:34:28 Down: 1.41MB/s / Disk: 2MB/s
(22:37:24) INFO:    [Legendary]:        Progress for Fallout: New Vegas: 0.84%/118.67MB/06:48:30 Down: 0.94MB/s / Disk: 9.93MB/s
```

### Steps to reproduce

1. Go to Library
3. Click on the game
4. Press the download button
5. Wait
6. See error

### Expected behavior

Downloaded

### Heroic Version

Latest Stable

### System Information

Heroic v2.7.1 flatpak from Flathub
Fedora 38

**Comment by arielj** [comment]
The memory message is a false error

I mean, it's an error, but Heroic already reads that output and updates the install command to add more memory so you don't have to do anything. You have this also in the logs:

```
Running command: /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary install 5daeb974a22a435988892319b3a4f476 --platform Windows --base-path /home/mks/Games/Heroic --with-dlcs --skip-sdl -y --max-shared-memory 5000
```

Note the added `--max-shaerd-memory 5000` at the end.

the issue you have is a known problem in 2.7.1 where download progress is not reported properly, but if you just wait it gets installed eventually

the error message should not be displayed (we should catch that before it's displayed but that's another issue)



tl;dr version: just wait and it will finish the installation

**Comment by mks-h** [comment]
OK, thanks!

**Comment by AlexBoXY** [comment]
Similar behavior here, but after about 5000MB download, the Heroic window turns completely white and all threads of legendary exit with a core dump, so I have to restart heroic once to continue the installation.

it is noticeable that before the threads crashes, a warning per thread appears in the log:

```
[DLManager] INFO: = Progress: 33.06% (6411/19394), Running for 00:20:40, ETA: 00:41:51
[DLManager] INFO:  - Downloaded: 5592.73 MiB, Written: 6104.39 MiB
[DLManager] INFO:  - Cache usage: 3895.00 MiB, active tasks: 32
[DLManager] INFO:  + Download	- 0.00 MiB/s (raw) / 0.00 MiB/s (decompressed)
[DLManager] INFO:  + Disk	- 0.00 MiB/s (write) / 0.00 MiB/s (read)
[FileWorker] WARNING: Writer queue empty!
[DLManager] INFO: = Progress: 33.06% (6411/19394), Running for 00:20:41, ETA: 00:41:53
[DLManager] INFO:  - Downloaded: 5592.73 MiB, Written: 6104.39 MiB
[DLManager] INFO:  - Cache usage: 3895.00 MiB, active tasks: 32
[DLManager] INFO:  + Download	- 0.00 MiB/s (raw) / 0.00 MiB/s (decompressed)
[DLManager] INFO:  + Disk	- 0.00 MiB/s (write) / 0.00 MiB/s (read)

```

**Comment by arielj** [comment]
closing, heroic already includes the legendary version that fixes this
